docs/getting-started/quickstart.md

```
@@ -57,6 +57,61 @@ docker compose up
 | 管理画面 | http://localhost:4000 | レポートの作成・管理 |
 | API | http://localhost:8000 | バックエンド API |
 
+## ローカル開発（Dockerを使わずに起動）
+
+Dockerを使わずに `apps/api` と `apps/admin` を個別に起動する場合は、以下を実行します。
+
+### 1. API 側の環境変数
+
+```bash
```

markdownlint-cli2 (0.20.0)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)